### PR TITLE
docs: improve stories

### PR DESCRIPTION
**Pull request by josokinas**
## Purpose

Improve some Storybook stories.

## Approach

Mainly to make sure defaults are selected, also set on a table (excluding "core" stories, as they should not have defaults).

Also making sure available values to be styled as in other stories - `"<someting>"`.

## Testing

On Storybook.

## Risks

N/A

### CODE REVIEW

File: packages/react/src/components/progress/progress.stories.tsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import React, { useEffect, useState } from 'react';
 import { Meta, omit, reactMatrix, Story } from '../../../../../docs';
 
 const size = ['regular', 'large'] as const;
+size.toString = () => size.map((value) => `"${value}"`).join('|');
```
Comment (rabelloo) [inline packages/react/src/components/progress/progress.stories.tsx:6]: This stringifier function could be a helper, since we need it in several stories.

**Review by rabelloo — APPROVED** [review]

File: packages/react/src/components/progress/progress.stories.tsx (hunk 1 above)
Comment (josokinas) [reply packages/react/src/components/progress/progress.stories.tsx]: Did create a helper for this, please confirm (last commit) if it's OK, or you'd suggest something different.

File: docs/helpers/docs.ts [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+ * Converts options array to a summary string.
+ *
+ * @param options Options array.
+ *
+ * @example
+ * optionsToSummary(['one', 'two'])
+ * // '"one" | "two"'
+ */
+export const optionsToSummary = (options: unknown) =>
+  (options as string[]).map((option) => `"${option}"`).join('|');
```
Comment (rabelloo) [inline docs/helpers/docs.ts:11]: Can't we type the input?

```suggestion
export const optionsToSummary = (options: readonly string[]) =>
  options.map((option) => `"${option}"`).join('|');
```

File: packages/react/src/components/progress/progress.stories.tsx (hunk 1 above)
Comment (rabelloo) [reply packages/react/src/components/progress/progress.stories.tsx]: Looks good, left a comment there.

**Pull request merged**